### PR TITLE
[reviewed-proto-renderer Part 4]: Link: repeated type. Slight `testingutils` refactoring

### PR DESCRIPTION
**Pull request by Marcial1234**
Part of #247 

Continues #291, now `InlineContent` is fully tested. Preparing for base nested type `Button`.

### CODE REVIEW

File: claat/proto-renderer/testing-utils/general.go [hunk 1]
```diff
@@ -26,15 +26,15 @@ func NewDummyProto() *tutorial.StylizedText {
 // renderingFunc is the tunction signature for output-format agnositic 'Render'
 type renderingFunc func(interface{}) (io.Reader, error)
```
Comment (Marcial1234) [inline claat/proto-renderer/testing-utils/general.go]: just refactoring in this file

File: claat/proto-renderer/html/templates-tests/stylizedtext_test.go [hunk 2]
```diff
@@ -9,7 +9,7 @@ import (
 )
 
 func TestRenderStylizedTextTemplateEscaping(t *testing.T) {
```
Comment (Marcial1234) [inline claat/proto-renderer/html/templates-tests/stylizedtext_test.go]: just refactoring in this file

File: claat/proto-renderer/html/templates-tests/paragraph_test.go [hunk 3]
```diff
@@ -9,7 +9,7 @@ import (
 )
 
 func TestRenderTestBlockTemplate(t *testing.T) {
```
Comment (Marcial1234) [inline claat/proto-renderer/html/templates-tests/paragraph_test.go]: just refactoring in this file

File: claat/proto-renderer/html/templates-tests/inlinecontent_test.go [hunk 4]
```diff
@@ -1,15 +1,74 @@
...
+			InProto:  testingutils.NewInlineContentLink(linkProto),
+			OutProto: linkProto,
+			Out:      weatherLinkOutput,
+			Ok:       true,
+		},
+	}
+	testingutils.RenderingIdendityTestBatch(html.Render, identityTests, t)
+}
+
+func TestRenderInlineContentTemplateIdentity(t *testing.T) {
```
Comment (Marcial1234) [inline claat/proto-renderer/html/templates-tests/inlinecontent_test.go]: This was moved from the end of the file

File: claat/proto-renderer/html/render.go [hunk 5]
```diff
@@ -27,7 +27,8 @@ func init() {
 }
 
 // Render returns the rendered HTML representation of a tutorial proto,
-// or the first error encountered rendering templates depth-first, if any
+// or the first error encountered rendering templates depth-first, if any.
+// Possible recursive desent follows proto definitions
```
Comment (cassierecher) [inline claat/proto-renderer/html/render.go]: descent*

File: claat/proto-renderer/html/templates-tests/inlinecontent_test.go [hunk 6]
```diff
@@ -1,15 +1,74 @@
...
+	"path/filepath"
 	"testing"
 
 	"github.com/googlecodelabs/tools/claat/proto-renderer/html"
 	"github.com/googlecodelabs/tools/claat/proto-renderer/testing-utils"
 	"github.com/googlecodelabs/tools/third_party"
 )
 
+func TestRenderInlineContentTemplateLinkFromFile(t *testing.T) {
+	linkFileRelDir := "src/github.com/googlecodelabs/tools/claat/proto-renderer/html/templates-tests/testdata/InlineContent/google_weather.txt"
```
Comment (cassierecher) [inline claat/proto-renderer/html/templates-tests/inlinecontent_test.go]: Make constant?

**Pull request closed without merging**